src/page/bank-payment.page.ts

```diff
@@ -4,7 +4,7 @@ export class BankPaymentPage {
   private confirmButton: ElementFinder;
 
   constructor () {
-    this.confirmButton = $('#cart_navigation > button > span');
+    this.confirmButton = $('#cart_navigation > button');
```

Comment:
I think that this doesn't make any difference with the original one, you can play a little bit more, try to use a different locator, for example, the text related with the button seems to be unique "I confirm my order"". Find a locator in where you can use that text.